Apps/WebClient/src/Server/Controllers/RobotsController.cs

```diff
@@ -40,7 +35,8 @@ public class RobotsController : Controller
         public RobotsController(IConfiguration configuration)
         {
             this.configuration = configuration;
-            this.robotsContent = this.configuration.GetValue("robots.txt", DefaultRobotsContent);
+            string robotsAssetContent = Common.Utils.AssetReader.Read(@"HealthGateway.WebClient.Server.Assets.Robots.txt");
```

Comment:
The asset should probably only be read once on controller initialization.

Reply:
since we create a transient controller its only initialized once per pod.

Reply:
Perfect, missed that!